docs/2.8.0/docs/kubernetes/01-introduction/overview.rst

```diff
@@ -7,57 +7,57 @@ Deploy on Kubernetes
+   The goal is a highly available, production-ready deployment. Such a deployment may have associated costs, depending on your cloud provider. If you need a solution for local development only, follow the `Getting Started with Daml <https://docs.daml.com/getting-started/index.html>`_ guide instead.
 
 .. image:: ../images/k8s-deployment.png
    :alt: Target Deployment
 
-For an overview of the Daml Enterprise architecture see `here <https://docs.daml.com/canton/about.html>`_.
+For an overview of the Daml Enterprise architecture, see `Introduction to Canton <https://docs.daml.com/canton/about.html>`_.
 
-* `Canton Domain <https://docs.daml.com/canton/architecture/overview.html#domain-entities>`_\. These services make up the *domain entities* of a Daml Enterprise deployment. Except for the *Sequencers* all of them achieve high-availability via active-passive clustering.
+* `Canton domain <https://docs.daml.com/canton/architecture/overview.html#domain-entities>`_\: These services make up the *domain entities* of a Daml Enterprise deployment. Except for the *sequencers*, all achieve high availability via active-passive clustering.
 
-* `Canton Participant <https://docs.daml.com/concepts/glossary.html#participant-node>`_\. This service provides access to the Daml Ledger for Daml applications. High-availability is achieved via active-passive clustering.
+* `Canton participant <https://docs.daml.com/concepts/glossary.html#participant-node>`_\: This service provides access to the Daml Ledger for Daml applications. You can achieve high availability through active-passive clustering.
 
-* `HTTP JSON API Service <https://docs.daml.com/json-api/index.html>`_\. This service exposes a JSON API for interacting with a *Participant*\ , allowing application writers to use a simpler version of the `Ledger API <https://docs.daml.com/app-dev/ledger-api.html>`_. High-availability is achieved via active-active clustering.
+* `HTTP JSON API service <https://docs.daml.com/json-api/index.html>`_\: This service exposes a JSON API for interacting with a *participant*\ , allowing application writers to use a simpler version of the `Ledger API <https://docs.daml.com/app-dev/ledger-api.html>`_. You can achieve high availability through active-active clustering.
 
-Note that all Helm charts referenced on the above target deployment architecture are available from `this repository <https://github.com/digital-asset/daml-helm-charts/tree/main>`_.
+You can find all of the Helm charts referenced by the target deployment architecture in the `Digital Asset Daml Helm Charts repository <https://github.com/digital-asset/daml-helm-charts/tree/main>`_. This may require a Daml Enterprise license.
```

Comment:
@hrischuk-da I added the sentence here about the Daml Enterprise license, but I'm not sure if this is the right place and the right wording. Any advice?

Reply:
I'm removing the license wording from this location at Miklos' advice. He suggests to add the disclaimer at the top of the "Deploy on Kubernetes" section. Looking for suggested wording and confirmation, @hrischuk-da.